.github/workflows/ci.yml

```diff
@@ -136,6 +137,13 @@ jobs:
           rm -rf dist
           python -m pip install --no-deps .
 
+      - name: Build and install periodictable
```

Comment:
I don't see why we can't just `pip install periodictable`?
This installs version `1.7.1` which seems numpy2 safe

Reply:
Good catch. I missed that. I only looked at the GitHub release list. I'll remove this.